packages/fast-usdc/src/exos/settler.js

```diff
@@ -293,22 +293,22 @@ export const prepareSettler = (
          *
          * @typedef {{
          *   txHash: EvmHash;
-         *   sender: NobleAddress;
+         *   nfa: NobleAddress;
          *   fullValue: NatValue;
          * }} SettlerTransferCtx
          */
         onFulfilled(_result, ctx) {
-          const { txHash, sender, fullValue } = ctx;
-          statusManager.forwarded(txHash, sender, fullValue);
+          const { txHash, nfa, fullValue } = ctx;
```

Comment:
`ctx` is part of the durable storage of the relevant vow, right? So this property name change would be a breaking change if this code were deployed, right?

Reply:
When we get to upgrade testing, I hope we manage to test upgrading with pending vows.